### ISSUE TITLE
Sand king ability bug insta kill

### ISSUE THREAD

**Issue by skiner7** [ValveSoftware/Dota2-Gameplay#10707]
### Ability name

Epicenter

### Description

when he buys the shard the joke begins

check ultimate never in coldown

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/141285255/99fde579-6884-4330-8717-284bb47f067f)

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/141285255/63556d73-ecaa-4724-b8e1-30fd18d446ad)

### Example Match ID (and possibly Timestamp)

7285611410 - 15:00

**Comment by TheChetu** [comment]
Yep, experiencing it right now in game. Instant killing everyone. Stunned him, instant kill.

**Comment by Kemmuli** [comment]
The bug happens when you buy shard before leveling up ultimate

**Comment by TheTuvean** [comment]
I´love every patch
